lib/Doctrine/DBAL/ColumnCase.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Doctrine\DBAL;
+
+/**
+ * Contains portable column case conversions.
+ */
+class ColumnCase
```

Comment:
Can we make these final, maybe?

Reply:
Yeah. I took it as is from your draft and though `final private function __construct()` is enough. But it only prohibits instantiation, not extension.